### PR TITLE
[ISSUE #3096] Actual value of parameter 'length' is always 'MessageUtils.SEQ_LENGTH'

### LINKED ISSUE #3096: [Enhancement] Actual value of parameter 'length' is always 'MessageUtils.SEQ_LENGTH'
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/MessageUtils.java line 185

### Describe the solution you'd like

e.g.
image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by jjbiggins**
Fixes #3096.

### Motivation

*Explain the content here.*
Requested Enhancement

*Explain why you want to make the changes and what problem you're trying to solve.*
Contribute to requested enhancement for MessageUtils.getRandomString() method.

### Modifications

*Describe the modifications you've done.*
Changed method signature to take zero arguments; then, updated the private getRandomString calls used throughout the MessageUtils class' various other methods.

### Documentation

- Does this pull request introduce a new feature? (yes / no) No
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented) N/A
- If a feature is not applicable for documentation, explain why? N/A
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/MessageUtils.java [hunk 1]
```diff
@@ -17,68 +17,63 @@
...
 import org.apache.eventmesh.common.protocol.SubscriptionItem;
 import org.apache.eventmesh.common.protocol.SubscriptionMode;
 import org.apache.eventmesh.common.protocol.SubscriptionType;
-import org.apache.eventmesh.common.protocol.tcp.Command;
-import org.apache.eventmesh.common.protocol.tcp.EventMeshMessage;
-import org.apache.eventmesh.common.protocol.tcp.Header;
 import org.apache.eventmesh.common.protocol.tcp.Package;
-import org.apache.eventmesh.common.protocol.tcp.Subscription;
-import org.apache.eventmesh.common.protocol.tcp.UserAgent;
+import org.apache.eventmesh.common.protocol.tcp.*;
```
Comment (mytang0) [inline eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/MessageUtils.java:29]: Please fix the checkStyle bug, do not import '.*'

File: eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/MessageUtils.java [hunk 2]
```diff
@@ -91,14 +86,14 @@ public static Package asyncMessageAck(Package in) {
 
     public static Package buildPackage(Object message, Command command) {
         final Package msg = new Package();
-        msg.setHeader(new Header(command, 0, null, generateRandomString(SEQ_LENGTH)));
+        msg.setHeader(new Header(command, 0, null, generateRandomString()));
         if (message instanceof CloudEvent) {
             final CloudEvent cloudEvent = (CloudEvent) message;
-            Preconditions.checkNotNull(cloudEvent.getDataContentType(), "DateContentType cannot be null");
+            Preconditions.checkNotNull(Objects.requireNonNull(cloudEvent.getDataContentType()), "DateContentType cannot be null");
```
Comment (mytang0) [inline eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/MessageUtils.java:92]: Isn't the non-null check repeated?

File: eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/MessageUtils.java (hunk 1 above)
Comment (jjbiggins) [reply eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/MessageUtils.java]: I must've had my IDE set to optimize imports; I corrected this, and commit.

File: eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/MessageUtils.java (hunk 2 above)
Comment (jjbiggins) [reply eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/MessageUtils.java]: Removed duplicated not null check

**Review by mytang0 — APPROVED** [review]
LGTM

**Pull request merged**